UI/signup.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>M-Tracker - SignUp</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./assets/css/account.css" />
+
+</head>
+    <body>
+        <br />
+        <br />
+        
+        <p align="center">
+            
+            <img src="./assets/images/logo.png" class="logo" />
+            <br />
+            <br />
+            <b>M-Tracker</b>
+        </p>
+  <div id="container">
+ 
+    <form action="dashboard_merchant/index.html" id="merchantForm">
+        
+        <p class="categoryPanelHeader"> Create An Account</p>
+
+        <input type="text" placeholder=" Fullname " id="text" required/>
+        <br />
+        <input type="email" placeholder=" Email " id="text" required/>
+        <br />
+        <input type="password" placeholder=" Password " id="text" required/>
+        <br />
+        <input type="password" placeholder="Confirm Password " id="text" required/>
+        <br />
+
+        <div align="center">
+            <button class="submitButton" id="submit" onclick=" ">Create Account</button>
+        </div>
+    </form>
+   
+    <p align="center" class="form-footer">
+                
+            Already have an account? <a href="login.html" class="formLinkColor">Login</a>
+      
+    </p>
+    <br />
+    <p align="center" class="form-footer">
+                
+            <a href="index.html" class="formLinkColor">Back Home</a>
+       
+     </p>
+
+
+</div>
+        <br />
```

Comment:
It is bad practise to **<br/>** it is better to use margins instead

Reply:
noted! I will retract from using break in my html codes. Thanks